components/service/glean/src/main/java/mozilla/components/service/glean/ping/BaselinePing.kt

```diff
@@ -0,0 +1,72 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.service.glean.ping
+
+import android.view.accessibility.AccessibilityManager
+import android.accessibilityservice.AccessibilityServiceInfo
+import android.content.Context
+import android.os.Build
+import mozilla.components.service.glean.metrics.GleanBaseline
+import mozilla.components.support.base.log.logger.Logger
+
+object BaselinePing : BaselinePingImplementation() {
```

Comment:
Everything looks good to me, except for this now :) In general, @pocmo suggested we should try to avoid singletons as much as possible, unless required by some design choice. In this case, since this is internal and handled by a singleton, I think we could do with a simple class.

Reply:
Okay, no more singleton.  I wasn't aware of trying to avoid them, will keep that in mind for the future 👍